content/news/playtest-20191020.md

```diff
@@ -0,0 +1,45 @@
+
+* Fixes for several issues with game loading/saving
+* Fixes for several crashes and a desync issue
+* Fixes for several glitches related to unit movement
+* Fixes for several mission related issues in Red Alert and Tiberian Dawn
+* Fixes for several compatibility issues with Linux and FreeBSD
+* Refinements to the new spectator UI
+* Further Red Alert balance adjustments
+
+We hope this will be the last playtest build, and if no further issues are found we expect a new release version in a few weeks time.
+Head on over to our [download page](/download/), try it out, and let us know if you find any issues or have any other feedback before it is too late!
+As always, the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/XXXX) is available on GitHub if you would like to find out more.
+
+<hr>
+
+Season 8 of the Red Alert Global League is well underway. Resident RAGL wrangler netnazgul has been posting weekly wrap-up reports on the forum ([Week 1](https://forum.openra.net/viewtopic.php?f=85&t=21029), [Week 2](https://forum.openra.net/viewtopic.php?f=85&t=21031)), covering the highlights and providing links to stream VODs for everybody who would like to keep up with the action.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20191020-ragl.gif" alt="Explosive RAGL action">
+
+RAGL Season 8 has been packed with explosive action.
+</div>
+
+<hr>
+
+We have been relatively quiet about OpenRA's Tiberian Sun mod lately, but this does not mean it has been abandoned. Development has been focused mainly on resolving low-level game engine issues, which has been slow due to the complexity of the work involved and relative lack of manpower.
+
+Examples of this work can be seen in [release-20180923](/news/release-20180923/) which included renderer changes that roughly doubled TS's in-game performance, in [release-20190314](news/release-20190314) which fixed a collection of game-breaking bugs with Nod's cloak generator technology, and in the upcoming release which significantly improves deployable units and aircraft.
+
+Deployable units were a big part of Tiberian Sun, with several combat units, support units, and even production facilities (in the Firestorm expansion) all having the ability to switch between mobile and stationary modes. OpenRA's unit-deploy code was originally written for the classic Construction Yards, and was poorly suited for the more dynamic gameplay cases used in Tiberian Sun.
+
+This code has been overhauled to add missing features and additional quality of life improvements. Deploy commands can now be queued between other orders (great for managing transports!), and deployed units can be ordered to undeploy and move to another location (including into a transport, or onto a service depot) as a single action. The risk of accidentally packing your Construction Yard with an accidental misclick has been reduced by requiring the force-move hotkey to be held while issuing these orders.
```

Comment:
"The risk of accidentally packing your Construction Yard with an accidental misclick [...]" has a double accidental, it's not too bad though I guess.